tensorflow/python/keras/layers/normalization_test.py

```diff
@@ -319,7 +319,7 @@ def test_v2_fused_attribute(self):
     norm = normalization_v2.BatchNormalization(fused=True)
     self.assertEqual(norm.fused, True)
     inp = keras.layers.Input(shape=(4, 4))
-    with self.assertRaisesRegex(ValueError, '4D input tensors'):
+    with self.assertRaisesRegex(ValueError, '4D or 5D input tensors'):
```

Comment:
Can you also add a test to this file? I think adding a test case to the `test_basic_batchnorm` is sufficient.

Reply:
Done